### ISSUE TITLE
View mode crashes in 3.1.0

### ISSUE THREAD

**Issue by adriaaah** [stefan-niedermann/nextcloud-notes#1042]
Thanks for the report. Should be fixed in `v3.0.1` - If it still occurs in this version or higher, just reopen the issue or create a new one.

_Originally posted by @stefan-niedermann in https://github.com/stefan-niedermann/nextcloud-notes/issues/1038#issuecomment-757472382_

---

Hello,

I've upgraded to 3.1.0 and #1038 is still reproducible to me. I tried to reopen that but couldn't.

**Comment by adriaaah** [comment]
I cannot reproduce it any more with v3.1.1

**Comment by stefan-niedermann** [comment]
Glad to hear that (though i didn't change anything related to this topic between `3.1.0` and `3.1.1`) - please feel free to reopen the issue if it occurs again.

**Comment by yzqzss** [comment]
```
App Version: 3.1.1
App Version Code: 3001001
App Flavor: fdroid

Files App Version Code: 20210122 (Beta)

---

OS Version: 3.10.84-perf-gfbbd0de(b5fb514e97)
OS API Level: 23
Device: A2015
Manufacturer: zte
Model (and Product): A2015 (A2015)

---

android.view.InflateException: Binary XML file line #23: Binary XML file line #23: Error inflating class it.niedermann.android.markdown.MarkdownViewerImpl
	at android.view.LayoutInflater.inflate(LayoutInflater.java:539)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
	at it.niedermann.owncloud.notes.databinding.FragmentNotePreviewBinding.inflate(FragmentNotePreviewBinding.java:65)
	at it.niedermann.owncloud.notes.edit.NotePreviewFragment.onCreateView(NotePreviewFragment.java:77)
	at androidx.fragment.app.Fragment.performCreateView(Fragment.java:2699)
	at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:320)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1199)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1368)
	at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:1446)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1509)
	at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:447)
	at androidx.fragment.app.FragmentManager.executeOps(FragmentManager.java:2181)
	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2004)
	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1959)
	at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1861)
	at androidx.fragment.app.FragmentManager$4.run(FragmentManager.java:413)
	at android.os.Handler.handleCallback(Handler.java:739)
	at android.os.Handler.dispatchMessage(Handler.java:95)
	at android.os.Looper.loop(Looper.java:148)
	at android.app.ActivityThread.main(ActivityThread.java:5461)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
Caused by: android.view.InflateException: Binary XML file line #23: Error inflating class it.niedermann.android.markdown.MarkdownViewerImpl
	at android.view.LayoutInflater.createView(LayoutInflater.java:645)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:764)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:835)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:838)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:838)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
	... 22 more
Caused by: java.lang.reflect.InvocationTargetException
	at java.lang.reflect.Constructor.newInstance(Native Method)
	at android.view.LayoutInflater.createView(LayoutInflater.java:619)
	... 31 more
Caused by: java.lang.NoClassDefFoundError: it.niedermann.android.markdown.markwon.-$$Lambda$MarkwonMarkdownViewer$N0nQp3p98292jlHz07UHK8yKrAg
	at it.niedermann.android.markdown.markwon.MarkwonMarkdownViewer.createMarkwonBuilder(MarkwonMarkdownViewer.java:89)
	at it.niedermann.android.markdown.markwon.MarkwonMarkdownViewer.<init>(MarkwonMarkdownViewer.java:68)
	at it.niedermann.android.markdown.markwon.MarkwonMarkdownViewer.<init>(MarkwonMarkdownViewer.java:63)
	at it.niedermann.android.markdown.MarkdownViewerImpl.<init>(MarkdownViewerImpl.java:18)
	... 33 more

```

**Comment by stefan-niedermann** [comment]
@yzqzss see #1056

**Comment by stefan-niedermann** [comment]
Will be fixed in `v3.1.2`.
